src/components/Setup/ProviderSelection.tsx

```diff
@@ -0,0 +1,23 @@
+import Col from 'react-bootstrap/Col';
+import Row from 'react-bootstrap/Row';
+import { Link } from 'react-router-dom';
+import { supportedProviders } from './Providers';
+
+export function ProviderSelection() {
+    return <>
+        <h3>Select Storage Type</h3>
+        <p>To connect to a repository or create one, select the preferred storage type:</p>
+        <Row>
+            <Col>
+                {supportedProviders.map(provider =>
+                    <Link key={provider.name}
+                        data-testid={'provider-' + provider.name}
```

Comment:
You can use the string interpolation syntax here:

```diff
- data-testid={'provider-' + provider.name}
+ data-testid={`provider-${provider.name}`}
```

I mention this because the line below this one uses that syntax and because I have seen other conversions from `"" + ""` syntax into `` `${}` ``  syntax in this PR; e.g. in the file, `src/utils/functions.ts`.